From e13ac7d420a8e5bc3d5f77acac05ab247974179f Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 23 Jan 2026 19:11:28 -0500
Subject: [PATCH] common/loggers/logger.go: Fix compile warnings

Fix these warnings

./logger.go:229:16: non-constant format string in call to (github.com/gohugoio/hugo/vendor/github.com/bep/logg.LevelLogger).Logf
./logger.go:245:15: non-constant format string in call to (github.com/gohugoio/hugo/vendor/github.com/bep/logg.LevelLogger).Logf
./logger.go:310:15: non-constant format string in call to (github.com/gohugoio/hugo/vendor/github.com/bep/logg.LevelLogger).Logf
./logger.go:322:16: non-constant format string in call to (github.com/gohugoio/hugo/vendor/github.com/bep/logg.LevelLogger).Logf

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
---
 common/loggers/logger.go | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/common/loggers/logger.go b/common/loggers/logger.go
index a013049f7701..c7dddbd2b907 100644
--- a/common/loggers/logger.go
+++ b/common/loggers/logger.go
@@ -226,7 +226,7 @@ func (l *logAdapter) Debugf(format string, v ...any) {
 }
 
 func (l *logAdapter) Debugln(v ...any) {
-	l.debugl.Logf(l.sprint(v...))
+	l.debugl.Logf("%s", l.sprint(v...))
 }
 
 func (l *logAdapter) Info() logg.LevelLogger {
@@ -242,7 +242,7 @@ func (l *logAdapter) Infof(format string, v ...any) {
 }
 
 func (l *logAdapter) Infoln(v ...any) {
-	l.infol.Logf(l.sprint(v...))
+	l.infol.Logf("%s", l.sprint(v...))
 }
 
 func (l *logAdapter) Level() logg.Level {
@@ -307,7 +307,7 @@ func (l *logAdapter) WarnCommand(command string) logg.LevelLogger {
 }
 
 func (l *logAdapter) Warnln(v ...any) {
-	l.warnl.Logf(l.sprint(v...))
+	l.warnl.Logf("%s", l.sprint(v...))
 }
 
 func (l *logAdapter) Error() logg.LevelLogger {
@@ -319,7 +319,7 @@ func (l *logAdapter) Errorf(format string, v ...any) {
 }
 
 func (l *logAdapter) Errorln(v ...any) {
-	l.errorl.Logf(l.sprint(v...))
+	l.errorl.Logf("%s", l.sprint(v...))
 }
 
 func (l *logAdapter) Errors() string {
-- 
2.52.0
